test/Microsoft.AspNet.Mvc.ModelBinding.Test/Validation/DefaultObjectValidatorTests.cs

```diff
@@ -472,19 +472,69 @@ public void NonRequestBoundModel_MarkedAsSkipped()
             validator.Validate(validationContext);
 
             // Assert
+            Assert.True(validationContext.ModelState.IsValid);
             var modelState = validationContext.ModelState["serviceProvider.TestService"];
             Assert.Empty(modelState.Errors);
             Assert.Equal(modelState.ValidationState, ModelValidationState.Skipped);
         }
 
+        [Fact]
```

Comment:
This should be a bigger set, should include all the types which you excluded from `SimpleTypeExcludeFilterTest`